### DISCUSSION TITLE
Mozilla Hubs for tabletop!

### DISCUSSION THREAD

**Discussion by Yoric** [mozilla/hubs#778] [General Questions & Discussion] [answered]
I believe that VR tabletop gaming could be a killer app of Mozilla Hubs: share a deck of cards, or a board game, or a few character sheets and dice, and get all your friend playing together even though they live on different continents.

**Comment by JollyGrin** [comment]
PLEASE PLEASE PLEASE!

I would be using Hubs everyday with friends if I could load in a few board games. Right now I load in a 9x9 Go board image and then use the pen or checker models as stones. Works okay enough for expermental play yet having broader support to make tabletop through WebVR would be GAMECHANGING

**Reply by elmau** [reply]
I think you can put spawners to play games like checkers, chess, go, ludo, and any board game that uses objects and dice.

**Comment by truedat101** [comment]
This is extremely relevant now.  Has anyone tried creating and importing a dice model?  Or chess pieces?

**Reply by truedat101** [reply]
Follow up to my own comment.  I tried just playing with some dice, and the physics for anything you spawn in via SketchFab or Poly will tend to fall through surfaces or float away.  The default physics for objects that get added to a scene make the objects very hard to use for anything other than flinging or floating.

**Reply by cvanmeurs** [reply]
If you hover over a 3D asset with the mouse cursor, hold down the spacebar and click on the icon that looks like a circle with an arrow pointing down (under the trash icon), it will enable "real" physics for the item.  Still not a great way to throw a dice, but I thought I'd mention that.

**Comment by JollyGrin** [comment]
I tried creating a game of Go by using an image of a board and pieces, but they occasionally fall through the board and easily get nudged out of the way.

**Comment by misslivirose** [comment, marked as answer]
Converting this to an issue since I'd love to get more discussion about what folks have in mind with this type of feature.

**Comment by Yoric** [comment]
My feeling is that both cards, dice and possibly chess-like pieces may be special enough that they'll need their own physics or non-physics (i.e. I'm ok with pressing a button to roll dice). It might even be necessary to further special-case for each game, as some games assign different meanings to cards depending on whether they're horizontal, vertical, reversed, etc.
